src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -63,19 +63,6 @@ void GlRenderer::flush()
 }
 
 
-void GlRenderer::currentContext()
-{
-#ifdef __EMSCRIPTEN__
-    auto targetContext = (EMSCRIPTEN_WEBGL_CONTEXT_HANDLE)mContext;
-    if (emscripten_webgl_get_current_context() != targetContext) {
-        emscripten_webgl_make_context_current(targetContext);
-    }
-#else
-    TVGERR("GL_ENGINE", "Maybe missing MakeCurrent() Call?");
```

Comment:
We can implement it here for portability. ThorVG users typically don't have deep knowledge of the ThorVG engine internals, so it's not ideal to let them manage the context themselves.